server/src/main/java/com/tencent/rss/server/LocalStorageChecker.java

```diff
@@ -132,5 +143,64 @@ boolean checkIsHealthy() {
       }
       return isHealthy;
     }
+
+    boolean checkStorageReadAndWrite() {
+      if (storage.isCorrupted()) {
+        return false;
+      }
+      File checkDir = new File(storageDir, "check");
+      try {
+        checkDir.mkdirs();
+        File writeFile = new File(checkDir, "test");
+        if (!writeFile.createNewFile()) {
+          return false;
+        }
+        try (FileOutputStream fos = new FileOutputStream(writeFile)) {
```

Comment:
There is too little data to check disk.
5 bytes lie within a single page.
Can you test multiple pages ?

Reply:
No need.